src/tests/Microsoft.PowerFx.Core.Tests/ExpressionTestCases/Abs.txt

```diff
@@ -9,6 +9,12 @@
 >> Abs(2)
 2
 
+>> Abs(2147483647) // testing a large postive number (+/- 2^31) the upper limit for an integer-type variable 
+2147483647
+
+>> Abs(9007199254740992) // testing a really large positive number (+/- 2^53-1) the upper limit for a long-type variable
```

Comment:
A "long-type" variable (assuming 'long' has its traditional meaning, a 64-bit integer value) could go all the way to 2^63 - 1, so this is not the upper limit for that type. 2^53 is the limit for the largest consecutive integer that represented without loss in a floating point type - which is what Power Fx uses if ParserOptions.NumberIsFloat (https://github.com/microsoft/Power-Fx/blob/main/src/libraries/Microsoft.PowerFx.Core/Parser/ParserOptions.cs#L26) is true.